### ISSUE TITLE
SystemError: Missing type object

### ISSUE THREAD

**Issue by suwangcompling** [huggingface/neuralcoref#133]
I loaded spacy with
```
import spacy
nlp = spacy.load('en_coref_lg')
```
and got the following error:
```
SystemError                               Traceback (most recent call last)
<ipython-input-4-4f770d791aa6> in <module>()
      1 import spacy
----> 2 nlp = spacy.load('en_coref_lg')
      3 # doc = nlp(u'My sister has a dog. She loves him.')
      4 
      5 # doc._.has_coref

/usr/local/lib/python3.7/site-packages/spacy/__init__.py in load(name, **overrides)
     19     if depr_path not in (True, False, None):
     20         deprecation_warning(Warnings.W001.format(path=depr_path))
---> 21     return util.load_model(name, **overrides)
     22 
     23 

/usr/local/lib/python3.7/site-packages/spacy/util.py in load_model(name, **overrides)
    112             return load_model_from_link(name, **overrides)
    113         if is_package(name):  # installed as package
--> 114             return load_model_from_package(name, **overrides)
    115         if Path(name).exists():  # path to model data directory
    116             return load_model_from_path(Path(name), **overrides)

/usr/local/lib/python3.7/site-packages/spacy/util.py in load_model_from_package(name, **overrides)
    133     """Load a model from an installed package."""
    134     cls = importlib.import_module(name)
--> 135     return cls.load(**overrides)
    136 
    137 

/usr/local/lib/python3.7/site-packages/en_coref_lg/__init__.py in load(**overrides)
     13     overrides['disable'] = disable + ['neuralcoref']
     14     nlp = load_model_from_init_py(__file__, **overrides)
---> 15     coref = NeuralCoref(nlp.vocab)
     16     coref.from_disk(nlp.path / 'neuralcoref')
     17     nlp.add_pipe(coref, name='neuralcoref')

SystemError: Missing type object
```

**Comment by suwangcompling** [comment]
Replacing `en_coref_lg` with `en_coref_md` I got a different error:
```
ValueError                                Traceback (most recent call last)
<ipython-input-1-2038b9fcc7a0> in <module>()
      1 import spacy
----> 2 nlp = spacy.load('en_coref_md')
      3 # doc = nlp(u'My sister has a dog. She loves him.')
      4 
      5 # doc._.has_coref

/usr/local/lib/python3.7/site-packages/spacy/__init__.py in load(name, **overrides)
     19     if depr_path not in (True, False, None):
     20         deprecation_warning(Warnings.W001.format(path=depr_path))
---> 21     return util.load_model(name, **overrides)
     22 
     23 

/usr/local/lib/python3.7/site-packages/spacy/util.py in load_model(name, **overrides)
    112             return load_model_from_link(name, **overrides)
    113         if is_package(name):  # installed as package
--> 114             return load_model_from_package(name, **overrides)
    115         if Path(name).exists():  # path to model data directory
    116             return load_model_from_path(Path(name), **overrides)

/usr/local/lib/python3.7/site-packages/spacy/util.py in load_model_from_package(name, **overrides)
    132 def load_model_from_package(name, **overrides):
    133     """Load a model from an installed package."""
--> 134     cls = importlib.import_module(name)
    135     return cls.load(**overrides)
    136 

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py in import_module(name, package)
    125                 break
    126             level += 1
--> 127     return _bootstrap._gcd_import(name[level:], package, level)
    128 
    129 

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap.py in _gcd_import(name, package, level)

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap.py in _find_and_load(name, import_)

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap.py in _find_and_load_unlocked(name, import_)

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap.py in _load_unlocked(spec)

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap_external.py in exec_module(self, module)

/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

/usr/local/lib/python3.7/site-packages/en_coref_md/__init__.py in <module>()
      4 from pathlib import Path
      5 from spacy.util import load_model_from_init_py, get_model_meta
----> 6 from en_coref_md.neuralcoref import NeuralCoref
      7 
      8 __version__ = get_model_meta(Path(__file__).parent)['version']

/usr/local/lib/python3.7/site-packages/en_coref_md/neuralcoref/__init__.py in <module>()
----> 1 from .neuralcoref import NeuralCoref

cymem.pxd in init en_coref_md.neuralcoref.neuralcoref()

ValueError: cymem.cymem.Pool has the wrong size, try recompiling. Expected 64, got 48
```

**Comment by samhavens** [comment]
Link to a [colab notebook](https://colab.research.google.com/drive/1NaCenozIa1GGXxmX60IHdljTQ03fIFZJ) demonstrating this

**Comment by mauryaland** [comment]
pip install spacy==2.0.12 should fixed it. It is a conflict problem with the package cymem that was in previous spacy releases in version 1.X and that is now in version 2.X.

**Comment by asadnawaz126** [comment]
@suwangcompling how did you fix this? I'm still getting the error

**Comment by petermartigny** [comment]
Hello!
I'm having the same issues (using spacy==2.0.12).

**Comment by suwangcompling** [comment]
> @suwangcompling how did you fix this? I'm still getting the error

I didn't. Upgrading to spacy==2.1.3 the problem automatically went away. 
I imagine neither digging into the source code nor waiting for a reply is ideal, if trying different versions doesn't work, I myself would go straight to an alternative tool (e.g. Stanford CoreNLP).

**Comment by thomwolf** [comment]
Hi, all these installation issue should be fixed in the latest release (4.0) and with SpaCy 2.1+
Feel free to open a new issue if you still experience installation problems.
